carbonserver/carbonserver/api/domain/team.py

```diff
@@ -0,0 +1,17 @@
+from carbonserver.database import schemas
+
+import abc
+
+
+class TeamInterface(abc.ABC):
+    @abc.abstractmethod
+    def save_organization(self, organization: schemas.OrganizationCreate):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_one_organization(self, organization_id):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_team_from_organizations(self, organization_id):
```

Comment:
same?

Reply:
I'm now realizing that might be WIP since it's all copy paste

Reply:
This is still WIP, but we could discuss main classes / Entities grammar indeed.
Emissions seems to be mainly used in group (we store a set of emissions in bulk, by run / experiment), but is it the same for the other entities ?
I guess Team / Organization classes would mainly be used in a singleton mode, at least to begin.